lwgenerate/gcp/gke.go

```diff
@@ -33,12 +35,13 @@ func (args *GenerateGkeTfConfigurationArgs) Generate() (string, error) {
 		return "", errors.Wrap(err, "invalid inputs")
 	}
 
-	requiredProviders, err := createRequiredProviders(args.UseExistingRequiredProviders)
+	requiredProviders, err := createRequiredProviders(args.UseExistingRequiredProviders, args.ExtraBlocksRootTerraform)
 	if err != nil {
 		return "", errors.Wrap(err, "failed to generate required providers")
 	}
 
 	gcpProvider, err := createGcpProvider(
+		map[string]interface{}{},
```

Comment:
For GKE, we likely need to pass provider args here as well or no?

Reply:
Done